config.go

```diff
@@ -81,6 +85,22 @@ func (cors *cors) applyCors(c *gin.Context) {
 	}
 }
 
+func (cors *cors) validateWildcardOrigin(origin string) bool {
+	for _, w := range cors.wildcardOrigins {
+		if w[0] == "*" && strings.HasSuffix(origin, w[1]) {
```

Comment:
the length of `w` is alway `>=1`? otherwise `index out of range`, right? @groovili